grpc_opa/authorizer.go

```diff
@@ -362,12 +362,19 @@ func (o OPAResponse) Obligations() (*ObligationsNode, error) {
 
 func redactJWT(jwt string) string {
 	parts := strings.Split(jwt, ".")
-	if len(parts) > 0 {
-		parts[len(parts)-1] = REDACTED
+	for i, _ := range parts {
```

Comment:
Sorry I think I wasn't very clear with my one of my comments.  Yes this redactJWT() fn redacts the signature, but ONLY the signature because we still want the full JWT header and body to be passed in the request to OPA.  If we redact the JWT body, OPA would not have any authz inputs to work on.  See how redactJWT() is being used in:
https://github.com/infobloxopen/atlas-authz-middleware/blob/767218dcf8dcd781b2176d2efdc6170c4d2ff665/grpc_opa/authorizer.go#L181
and how `opaReq` is being passed to the request to OPA.

Redaction of all three JWT header/body/signature should only be done for log output.

Reply:
Yeah I misunderstood you but at the beginning I ensured that we copy the struct and modify it because that is only needed for debugging and should not affect the rest of the code but got confused with redactJWT that is done not for debug but in general. Now I know the biology of jwt and it makes sense.

Reply:
`for i := range parts` is sufficient, don't need (and go fmt will complain) `for i, _ := range parts`

Reply:
Yeah it is redundant, all removed